### PR TITLE
add F5 IP to whitelists

### PR DESCRIPTION
**Pull request by Ianrfisher**
### Change description ###
Add the IP of the F5 VPN to the juror.{env}.apps.hmcts.net AFD rules

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: environments/demo/demo.tfvars [hunk 1]
```diff
@@ -585,6 +585,7 @@ frontends = [
               "20.49.214.199/32",
               "20.49.214.228/32",
               "193.237.208.55/32",
+              "20.58.23.145/32",
```
Comment (thomast1906) [inline environments/demo/demo.tfvars:588]:
```suggestion
              "20.58.23.145/32"
```

File: environments/ithc/ithc.tfvars [hunk 2]
```diff
@@ -167,6 +167,7 @@ frontends = [
               "20.49.214.199/32",
               "20.49.214.228/32",
               "193.237.208.55/32",
+              "20.58.23.145/32",
```
Comment (thomast1906) [inline environments/ithc/ithc.tfvars:170]:
```suggestion
              "20.58.23.145/32"
```

File: environments/stg/stg.tfvars [hunk 3]
```diff
@@ -839,6 +839,7 @@ frontends = [
               "20.49.214.199/32",
               "20.49.214.228/32",
               "193.237.208.55/32",
+              "20.58.23.145/32",
```
Comment (thomast1906) [inline environments/stg/stg.tfvars:842]:
```suggestion
              "20.58.23.145/32"
```

File: environments/test/test.tfvars [hunk 4]
```diff
@@ -559,6 +559,7 @@ frontends = [
               "20.49.214.199/32",
               "20.49.214.228/32",
               "193.237.208.55/32",
+              "20.58.23.145/32",
```
Comment (thomast1906) [inline environments/test/test.tfvars:562]:
```suggestion
              "20.58.23.145/32"
```

**Review by thomast1906 — CHANGES_REQUESTED** [review]
see comments

**Review by thomast1906 — APPROVED** [review]

**Comment by Ianrfisher** [comment]
> see comments

Changes accepted - thanks

File: environments/demo/backend_lb_config.yaml [hunk 5]
```diff
@@ -70,10 +70,7 @@ gateways:
         component: api
         ssl_enabled: true
 
-     # Juror (bureau added to allow access via AppGw for developers using F5 VPN)
```
[the remark this answers is not in the document]
Comment (Ianrfisher) [reply environments/demo/backend_lb_config.yaml:73]: yes - this was added recently by me - to test an alternative DNS route that bypassed AFD - but it didn't work without messing about with multiple ingress hosts - which I decided was a bad idea. Hence adding F% VPN IP to whitelist instead

**Pull request merged**